### ISSUE TITLE
Shortkeys now requires more permissions?

### ISSUE THREAD

**Issue by duquet** [crittermike/shortkeys#624]
I am a huge fan of Shortkeys but it has now been disabled in Chrome Extensions for requiring more permissions such as read and change all your data on all websites (please see attached). Is there a version of ShortKeys that doesn't require this?

**Comment by minerharry** [comment]
I don't think so, it uses those permissions to inject a listener script into websites to process the hotkeys.
